cli/data.go

```diff
@@ -495,7 +495,15 @@ func downloadBinary(ctx context.Context, client datapb.DataServiceClient, dst st
 			req.Header.Add("key", apiKey.KeyCrypto)
 		}
 
-		res, err := httpClient.Do(req)
+		var res *http.Response
+		for count := 0; count < maxRetryCount; count++ {
+			res, err = httpClient.Do(req)
```

Comment:
Do we care at all about retrying on any non-2xx status codes? The error will only be non-nil on failure to talk HTTP or due to client policy. I think this is an improvement either way, so LGTM but just pointing out :)

Reply:
oh, good point! looking into this.

Reply:
done now :)